README.md

```diff
@@ -113,7 +113,7 @@ Please be aware that this sample application includes a publicly accessible Appl
 5. Create Canaries and SLOs
 
    ``` shell
-   endpoint=$(kubectl get ingress -o json  --output jsonpath='{.items[0].status.loadBalancer.ingress[0].hostname}')
+   endpoint="http://$(kubectl get ingress -o json  --output jsonpath='{.items[0].status.loadBalancer.ingress[0].hostname}')"
```

Comment:
Can you please run the `Deploy via Terraform` steps up to and including this step and provide the output? I want to see that this will successfully run.

Reply:
See https://github.com/aws-observability/application-signals-demo/pull/51#issuecomment-2318342219 - LGTM